client/signup/steps/domains/utils.js

```diff
@@ -33,18 +33,13 @@ export function getExternalBackUrl( source, sectionName = null ) {
 /**
  * Check if we should use multiple domains in domain flows.
  */
-export function shouldUseMultipleDomainsInCart( flowName, currentUser ) {
-	const enabledFlows = [ 'domain' ];
-	const enabledFlowsUnderFlag = [ 'onboarding' ];
+export function shouldUseMultipleDomainsInCart( flowName ) {
+	const enabledFlows = [ 'domain', 'onboarding' ];
 
+	// The onboarding E2E tests are not yet updated to handle multiple domains.
 	if ( isE2ETest() ) {
 		return false;
 	}
```

Comment:
I see, we'll need another PR to remove this after deploying the current one?

Reply:
I believe we need to update the E2E tests first before we can remove this check. Issue here: https://github.com/Automattic/dotcom-forge/issues/4371